docs/integrations/sources/google-sheets.md

```diff
@@ -44,12 +50,15 @@ To set up Google Sheets as a source in Airbyte Open Source:
    - To authenticate your Google account via OAuth, enter your Google application's [client ID, client secret, and refresh token](https://developers.google.com/identity/protocols/oauth2).
    - To authenticate your Google account via Service Account Key Authentication, enter your [Google Cloud service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys) in JSON format. Make sure the Service Account has the Project Viewer permission. If your spreadsheet is viewable by anyone with its link, no further action is needed. If not, [give your Service account access to your spreadsheet](https://youtu.be/GyomEw5a2NQ%22).
 6. For **Spreadsheet Link**, enter the link to the Google spreadsheet. To get the link, go to the Google spreadsheet you want to sync, click **Share** in the top right corner, and click **Copy Link**.
+7. For **Row Batch Size**, define the number of records you want the Google API to fetch at a time. The default value is 200.
+8. For **Columns Name Conversion**, if set as true column names will be transformed. Examples: My_Name -> my_name, My Name -> my_name, MyName -> my_name, My123name -> my_123_name, привіт світ -> privit_svit.
 
 ### Output schema
 
 Each sheet in the selected spreadsheet is synced as a separate stream. Each selected column in the sheet is synced as a string field.
-
-**Note: Sheet names and column headers must contain only alphanumeric characters or `_`, as specified in the** [**Airbyte Protocol**](../../understanding-airbyte/airbyte-protocol.md). For example, if your sheet or column header is named `the data`, rename it to `the_data`. This restriction does not apply to non-header cell values.
+    :::note
+    Sheet names and column headers must contain only alphanumeric characters or `_`, as specified in the** [**Airbyte Protocol**](../../understanding-airbyte/airbyte-protocol.md). For example, if your sheet or column header is named `the data`, rename it to `the_data`. This restriction does not apply to non-header cell values.
```

Comment:
thank you for pointing this out. source has convert for sheet columns(Convert Column Names to SQL-Compliant Format in config), but doesn't have convert for sheet names. the source works normally if the names of the sheets have not been converted to the standard. I’ve tested it.
From the source side we do not need to change the source code (we can convert sheet names too but it will be a breaking change). the Airbyte protocol says that destinations are responsible for cleaning these names to make them valid table and column names in their respective data stores. 
So I removed this note from the documentation.